azurerm/internal/services/keyvault/tests/key_vault_certificate_issuer_data_source_test.go

```diff
@@ -0,0 +1,40 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+)
+
+func TestAccDataSourceAzureRMKeyVaultCertificateIssuer_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "data.azurerm_key_vault_certificate_issuer", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { acceptance.PreCheck(t) },
+		Providers: acceptance.SupportedProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceAzureRMKeyVaultCertificateIssuer_basic(data),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(data.ResourceName, "account_id", "test-account"),
+					resource.TestCheckResourceAttr(data.ResourceName, "provider_name", "DigiCert"),
+					resource.TestCheckResourceAttr(data.ResourceName, "password", "test"),
```

Comment:
Are there any other exported attributes that can be collected and checked from the resource used?